### PR TITLE
Feature/status

### PR DESCRIPTION
**Pull request by Lauraducky**

### CODE REVIEW

File: src/PockyBot.NET/Services/Triggers/Status.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                }
+            }
+
+            return Task.FromResult(new Message
+            {
+                Text = response.ToString()
+            });
+        }
+
+        private Dictionary<bool, List<Persistence.Models.Peg>> GroupPegsByValidity(PockyUser pockyUser)
```
Comment (jimcullenaus) [inline src/PockyBot.NET/Services/Triggers/Status.cs]: I really love this approach!

File: README.md [hunk 2]
```diff
@@ -153,13 +159,16 @@ sent directly to the bot. In this readme, mentions will be identified by
...
+-   `@PockyBot ping` — verify that the bot is alive.
+-   `@PockyBot peg @Person <reason>` — give someone a peg.
+-   `@PockyBot status` — get the list of pegs you have given.
 
 #### Direct Message Commands
 
 PockyBot can be messaged directly with certain commands.
 
-- `ping`
+-   `ping`
```
[the remark this answers is not in the document]
Comment (Lauraducky) [reply README.md]: It's to do with RemarkLint, the tool that it uses to validate markdown. We can turn this rule off or try to configure it if we like

https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-list-item-indent

File: src/PockyBot.NET/Services/Triggers/Status.cs [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        public string Command => Commands.Status;
+
+        public bool DirectMessageAllowed => true;
+
+        public bool CanHaveArgs => false;
+
+        public string[] Permissions => Array.Empty<string>();
+
+        public Task<Message> Respond(Message message)
```
Comment (gillianmilne) [inline src/PockyBot.NET/Services/Triggers/Status.cs]: maybe break this down a bit further

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jimcullenaus) [reply README.md]: [Based on this](https://github.com/remarkjs/remark-lint/issues/178), as well as the detail in your link, this seems like a poorly-implemented/poorly-designed feature IMO. Better to disable that option I think.

File: src/PockyBot.NET/Services/Triggers/Status.cs [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        private static string GetPegsLeftText(PockyUser pockyUser, int limit, int pegsGiven = 0)
+        {
+            var amountLeft = pockyUser?.HasRole(Roles.Unmetered) ?? false ? "unlimited" : (limit - pegsGiven).ToString();
+            return $"You have {amountLeft} pegs left to give.";
+        }
+
+        private static string GetValidPegsSentText(List<Persistence.Models.Peg> validPegs)
+        {
+            
+            if (validPegs.Count > 0)
```
Comment (craigngu) [inline src/PockyBot.NET/Services/Triggers/Status.cs]: Minor: Personally prefer validPegs.Any() over count > 0

**Pull request merged**